### PR TITLE
Added class support

### PR DESCRIPTION
**Pull request by straps**
Classes are very useful when you need to group items and use them programmatically.

Merge this please, it just works, thanks

### CODE REVIEW

**Comment by codedread** [comment]
Do the toolbar components wrap properly?

File: editor/svg-editor.js [hunk 1]
```diff
@@ -2991,7 +2992,7 @@ TODOS
 					return false;
 				}
 
-				if (attr !== 'id') {
+				if (attr !== 'id' && attr!=='class') {
```
Comment (codedread) [inline editor/svg-editor.js]: nit: can you please space things out here like the code around it:

  attr !== 'class'

**Comment by straps** [comment]
What do you mean? HTML structure is the same as ID field...

File: editor/svg-editor.js (hunk 1 above)
Comment (straps) [inline editor/svg-editor.js]: done, commit ce009a4

**Pull request merged**